tests/test_preproc.py

```diff
@@ -300,7 +302,8 @@ def test_trial_align_events(self):
             [0. , 0.1],
             [0. , 0. ],
```

Comment:
i don't understand why there are some [0., 0.] rows in here, should they all be [0., 0.1]?

Reply:
The [0,0] rows are where there are consecutive event codes to align to. Should probably be fixed though.